test/AspNetCore.Mvc.HttpActionResults.ClientError.Test/RequestUriTooLongResultTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace AspNetCore.Mvc.HttpActionResults.ClientError.Test
+{
+    using AspNetCore.Mvc.HttpActionResults.ClientError.Test.Common;
+
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+
+    using Xunit;
+
+    public class RequestUriTooLongResultTests : BaseHttpResultTests
+    {
+        [Fact]
+        public async void RequestUriTooLongResultShouldSetStatusCodeCorrectly()
+        {
+            // Arrange
+            var fakeContext = this.CreateFakeActionContext();
+            var result = new RequestUriTooLongResult();
+
+            // Act
+            await result.ExecuteResultAsync(fakeContext);
```

Comment:
Why are you executing the result on this test? I believe the `StatusCode` property is enough to be checked?